yogstation/code/modules/mob/living/carbon/human/species_types/preternis/preternis.dm

```diff
@@ -16,14 +16,14 @@ adjust_charge - take a positive or negative value to adjust the charge level
 	attack_verb = "assault"
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/synthmeat
 	toxic_food = NONE
-	brutemod = 1.25 //Have you ever punched a metal plate?
-	burnmod = 1.5 //Computers don't like heat
-	coldmod = 0.8 //Computers like cold, but their lungs may not
-	heatmod = 1.75 //Again, computers don't like heat
-	speedmod = 0.1 //Metal legs are heavy and slow
+	brutemod = 0.8 //Robusto plasma plating
+	burnmod = 0.9 //Circuitry isn't happy with the heat, even if plasma plating lessens it
+	coldmod = 0.8 //Cold can help to regulate, but it's not perfect
+	heatmod = 0.9 //Again, circuitry isn't happy, but the plating they have on the outside can protect a little
+	speedmod = 1 //Effectively a plasteel body, big and slow
```

Comment:
a speed mod of 1 (which iirc is half?) might be a bit brutally slow

Reply:
It is. I’m tempted to reduce it but I would rather go hard on the main downside at first when they are effectively better humans in almost every other right